disk/disk_aix_nocgo.go

```diff
@@ -79,3 +81,145 @@ func PartitionsWithContext(ctx context.Context, all bool) ([]PartitionStat, erro
 func getFsType(stat unix.Statfs_t) string {
 	return FSType[int(stat.Vfstype)]
 }
+
+func UsageWithContext(ctx context.Context, path string) (*UsageStat, error) {
```

Comment:
I think the functions added to `disk_aix_nocgo.go` would work with cgo as well. How about adding them to `disk_aix.go` instead? This way, it will work with both nocgo and cgo.

Reply:
They definitely will, the only reason I did it this way was because there is a better method to do it in CGO, I just don't fully understand it, so I didn't implement it.


Since your idea makes sense, I'll definitely do that when I'm back at the laptop tomorrow.